_documentation/05-User-Manual.md

```diff
@@ -131,9 +131,60 @@ ___
 
     ![Webp net-resizeimage (10)](https://user-images.githubusercontent.com/43112419/97261802-9c5bbd00-1845-11eb-9618-5907372c0755.png)
 
-    
+    Many manufacturers have options that cannot be translated into IPP attributes. So web interface provides the option to set up these options in this segment.
```

Comment:
Many **manufacturer's devices** have options that cannot be translated into IPP attributes. So **the** web interface provides the **possibility** to set up these options.